src/trendlines/utils.py

```diff
@@ -96,20 +104,21 @@ def build_jstree_data(metrics):
 
     Parameters
     ----------
-    metrics : list of str
-        The metric names to display. Note that this is *not* the Metric
-        objects themselves, but rather a simple list of strings (metric
-        names). Take the results of :func:`db.get_metrics`::
+    metrics : list of :class:`orm.Metric` objects
```

Comment:
I think these links need the `trendlines` name:

```
:class:`trendlines.orm.Metric`
```

We could probably shorten the resulting names with `~` and it would still make sense:


```
:class:`~trendlines.orm.Metric`
```